docs/ibc/middleware/integration.md

```diff
@@ -46,18 +46,18 @@ scopedKeeperCustom2 := capabilityKeeper.NewScopedKeeper("custom2")
 // initialize base IBC applications
 // if you want to create two different stacks with the same base application,
 // they must be given different scopedKeepers and assigned different ports.
-transferIBCModule := transfer.NewIBCModule(transferKeeper, scopedKeeperTransfer)
-customIBCModule1 := custom.NewIBCModule(customKeeper, scopedKeeperCustom1, "portCustom1")
-customIBCModule2 := custom.NewIBCModule(customKeeper, scopedKeeperCustom2, "portCustom2")
+transferIBCModule := transfer.NewIBCModule(transferKeeper)
```

Comment:
I am actually not sure about this part... Looking at the code, the `NewIBCModule` functions don't take a scoped keeper or port...

Reply:
I think its fine to leave it out here. As far as I know its the transfer `Keeper` which takes the scoped keeper